src/app/styles/modules/_colors.scss

```diff
@@ -15,6 +15,8 @@ $dark-grey: #ECECEC;
 $grey-background-alpha: rgba(0, 0, 0, 0.7);
 $orange: #DC5034;
 $dark-orange: #EBBB67;
+$light-grey: #75787B;
+$blue-nav: #1E1656;
```

Comment:
Does this refer to navy blue or blue used for navigation?
This file should just contain color names and should not refer to any component.
If it is navy blue then rename it to navy-blue.